core/renderers/common/marker_svg.js

```diff
@@ -323,10 +411,12 @@ Blockly.blockRendering.MarkerSvg.prototype.positionBlock_ = function(
 /**
  * Position the marker for an input connection.
  * Displays a filled in puzzle piece.
- * @param {!Blockly.RenderedConnection} connection The connection to position marker around.
+ * @param {!Blockly.RenderedConnection} connection The connection to position
+ *     marker around.
  * @private
```

Comment:
Why is this one private and the rest protected.

Reply:
There were reasons but rethinking it seems better to make them all protected.